packages/lesswrong/components/posts/AddToCalendar/makeUrls.ts

```diff
@@ -56,17 +56,32 @@ const makeICSCalendarUrl = (event: CalendarEvent) => {
     "BEGIN:VEVENT"
   ];
 
+  // The ICS format requires escaping of certain characters
+  const escapeText = (text: string) => text.replace("\\", "\\\\").replace("\n", "\\n").replace(",", "\\,").replace(";", "\\;");
```

Comment:
I think this will only catch the first instance.
```suggestion
const escapeText = (text: string | null) => {
  if(!text) {
    return null;
  }
  return text.replace(/\\/g, "\\\\").replace(/\n/g, "\\n").replace(/,/g, "\\,").replace(/;/g, "\\;");
}
```